python/src/scipp/plot/figure3d.py

```diff
@@ -405,3 +404,7 @@ def update_log_axes_buttons(self, *args, **kwargs):
         """
         if self.toolbar is not None:
             self.toolbar.update_log_axes_buttons(*args, **kwargs)
+
+    def update_norm_button(self, *args, **kwargs):
+        if self.toolbar is not None:
+            self.toolbar.update_norm_button(*args, **kwargs)
```

Comment:
Same as in `figure.py`? Should there be a common base class?

Reply:
I already have quite a few base classes, I'm not sure I want to make a new one just for one function. I'm starting to feel that the more inheritance I add the less flexible everything becomes.